README.md

```diff
@@ -196,6 +196,7 @@ selectedLocation: InitialLocationModel = {
 <aui-location-picker
     [baseUrl]="baseUrl"
     [defaultZoom]="defaultZoom"
+    [changeZoomLevelToSelectZoom]="changeZoomLevelToSelectZoom"
```

Comment:
Isn't it more logical to call this property `changeZoomLevelToOnSelectZoom`?
What do you think, @jsebrech ?

Reply:
Yes, the name is not at all clear. Why is this property needed at all? Can't you make onSelectZoom nullable so that you can set a null value to disable it?

Reply:
Made onSelectZoom nullable, property was indeed unnecessary.